src/Directory.Build.props

```
@@ -17,7 +17,7 @@
 
 		<Product>$(AssemblyName) ($(TargetFramework))</Product>
 		<DefaultLanguage>en-US</DefaultLanguage>
-		<LangVersion>9.0</LangVersion>
+		<LangVersion>10.0</LangVersion>
```

Personally I've started to use the "latest":

```suggestion
		<LangVersion>latest</LangVersion>
```